source/Item.hx

```diff
@@ -0,0 +1,86 @@
+package;
+import flixel.FlxG;
+import flixel.FlxObject;
+import flixel.FlxSprite;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+import haxe.Timer;
+
+
+/**
+ * ...
+ * @author Cameron Yuan
+ */
+class Item extends FlxSprite
+{
+	private var player_trace:Player;
```

Comment:
For the direction, that could just be passed into some function of either `Item` or `Sword` whenever the player moves left or right.  Something like 
`public function setFacing(newFacing:Int) {
  facing = newFacing;
}`
and use that in `Player` by replacing the current `facing = FlxObject.LEFT;` (and the right one) with
`if (weilding) {
  equipped_item.setFacing(FlxObject.LEFT);
}`

You should be able to access the `Player`'s coordinates from inside `Item` or `Sword` (I'm leaning toward the former) with `(cast (FlxG.state, PlayState)).player.x`  You could also leave the facing part of `Player` as is and just access it this way.